mne/beamformer/tests/test_dics.py

```diff
@@ -669,4 +674,52 @@ def test_tf_dics(_load_forward, idx):
     assert np.all(np.isnan(stcs[0].data))
 
 
-run_tests_if_main()
+def _cov_as_csd(cov, info):
+    rng = np.random.RandomState(0)
+    assert cov['data'].ndim == 2
+    assert len(cov['data']) == len(cov['names'])
+    # we need to make this have at least some complex structure
+    data = cov['data'] + 1e-1 * _rand_csd(rng, info)
+    assert data.dtype == np.complex128
+    return CrossSpectralDensity(_sym_mat_to_vector(data), cov['names'], 0., 16)
+
+
+# Just test free ori here (assume fixed is same as LCMV if these are)
+@pytest.mark.parametrize(
+    'reg, pick_ori, weight_norm, use_cov, depth, lower, upper, real_filter', [
+        (0.05, None, 'unit-noise-gain-invariant', False, None, 26, 28, False),
+        (0.05, None, 'unit-noise-gain-invariant', True, None, 40, 42, False),
```

Comment:
How did you pick `upper` and `lower` percentages? Copied from LCMV?

Reply:
Yep, all values here match LCMV. Really it should stay that way, so once we settle on whether or not to change when `real_filter` happens I'll add a comment mentioning that they should stay in sync. Or maybe I'll just define different subset tuples in LCMV and use them here to make sure they match both places... I'll see if that stays readable.

Reply:
Yes, agreed, they should match. How did we come up for them for the LCMV again?

Reply:
They are just empirical -- they reflect the values that we produce and have produced for a while. The idea is that we should only have to change them when we find a bug -- and hopefully the values generally go up when we do this. And similarly, they help us see when we've broken something in our code because they are reasonably sensitivity to breaking changes.

Reply:
Okay, that's what I thought. So they would presume the implementation to be correct ... :-) Well I guess for a lot we have checks that things actually _are_ correct as well. 
Speaking of which, once we decided upon the `real` orientation selection issue, I can push the external tests (against FieldTrip) for the DICS.